pegasus/sites.v3/code.org/public/teacher-dashboard/index.haml

```diff
@@ -35,12 +36,17 @@ title: <%= I18n.t('dashboard_landing_title').inspect %>
 %link(rel='stylesheet' type='text/css' href='/shared/css/teacher-dashboard.css')
 %link(rel='stylesheet' type='text/css' href='/shared/css/user-progress.css')
 
+- js_locale = request.locale.to_s.downcase.tr('-', '_')
```

Comment:
Pretty confident, since this is how it is done for tutorialExplorer on code.org/learn: https://github.com/code-dot-org/code-dot-org/blob/staging/pegasus/sites.v3/code.org/public/learn/index.haml#L82-L84